### ISSUE TITLE
Data File Too short Error

### ISSUE THREAD

**Issue by CommunistMeth** [bijx/Zeus-Music-Mod-Generator#2]
So basically I followed the insturctions, 30 songs, successful on every step and then as I packed and uploaded it, when I load the mod it says this error? Anyway you can assist me ? You can hit me up on discord if it is faster.

**Comment by bijx** [comment]
@CommunistMeth Yep! Could you post a screenshot of the error on this thread (or DM me) here so I can see what's going on?

**Comment by CommunistMeth** [comment]
Will do when I get home!!

**Comment by bijx** [comment]
@CommunistMeth Did you ever DM me the screenshot, or is it not causing a problem anymore?

**Comment by OshidaBCF** [comment]
So i've been having the same issue, and it seems that this happen if any "special" characters is in the files name

if you open the pso file, you can see a list of objects, if the names contrain "?" then that mean it was a special character (Russian or Japanese for exemple)

Is this a limitation of the tool or limitation of Arma ?
I don't want to spend hours renaming russian and japanese title to english.

**Comment by bijx** [comment]
> So i've been having the same issue, and it seems that this happen if any "special" characters is in the files name
> 
> if you open the pso file, you can see a list of objects, if the names contrain "?" then that mean it was a special character (Russian or Japanese for exemple)
> 
> Is this a limitation of the tool or limitation of Arma ? I don't want to spend hours renaming russian and japanese title to english.

This sounds like a limitation of Arma as it doesn't really play well with non-alphanumeric characters. When I get around to it, I will add a feature to the tool that automatically removes illegal characters, but for now it is up to users to make sure the files are named correctly. Thanks for adding context!

**Comment by OshidaBCF** [comment]
Okay i managed to make my script works

files names cannot contain special characters, but the "song" name can

what i did is just rename every file to a number but keep the name in the mod
I made a simple pyscript if you want to check
https://github.com/OshidaBCF/Arma-3-Music-Mod
